Ошибка: HTTPConnectionPool(host='81', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000002027ABC4B50>: Failed to resolve '81' ([Errno 11001] getaddrinfo failed)"))